### PR TITLE
feat(operator): add gnark plonk verifier

### PR DESCRIPTION
**Pull request by entropidelic**
This PR adds consensys gnark PLONK verifier in the operator

### CODE REVIEW

File: operator/pkg/operator.go [hunk 1]
```diff
@@ -73,14 +79,90 @@ func (o *Operator) Start(ctx context.Context) error {
...
+	if err != nil {
+		panic("Could not read PLONK verifying key from bytes")
+	}
+
+	err = plonk.Verify(proof, verificationKey, pubInput)
+	if err != nil {
+		return false
+	} else {
+		return true
+	}
```
Comment (taturosati) [inline operator/pkg/operator.go]:
```suggestion
        return err == nil
```

simplify

**Review by taturosati — APPROVED** [review]
lgtm. tried invalid proof as well & got false

File: operator/pkg/operator.go (hunk 1 above)
Comment (entropidelic) [reply operator/pkg/operator.go]: thanks, done!

**Review by MauroToscano — APPROVED** [review]
It may be nicer to keep printing the task hash. Signature data is not very useful

**Pull request merged**